masterlist.yaml

```diff
@@ -6158,75 +5736,47 @@ plugins:
             text: '[Friendlier Taverns with Baths and Barns SSE](https://www.nexusmods.com/skyrimspecialedition/mods/1399/) is incompatible with JK''s Skyrim. Please use [Friendlier Taverns](https://www.nexusmods.com/skyrimspecialedition/mods/2506) instead.'
           - lang: de
             text: '[Friendlier Taverns with Baths and Barns SSE](https://www.nexusmods.com/skyrimspecialedition/mods/1399/) ist inkompatibel mit JK''s Skyrim. Bitte nutzen Sie stattdessen [Friendlier Taverns](https://www.nexusmods.com/skyrimspecialedition/mods/2506).'
-      - <<: *hasPluginPatch
-        subs:
-          - 'Better City Entrances'
-          - '[JK''s Skyrim SE Better City Entrances Patch](https://www.nexusmods.com/skyrimspecialedition/mods/6289/)'
+      - <<: *patchProvided
+        subs: [ 'Better City Entrances' ]
         condition: 'active("Better City Entrances - All in one - SSE.esp") and not active("JKs Skyrim_Better City Entrances_Patch.esp")'
-      - <<: *hasPluginPatch
-        subs:
-          - 'Better Docks'
-          - '[JK''s Skyrim SE Better Docks Patch](https://www.nexusmods.com/skyrimspecialedition/mods/6289/)'
+      - <<: *patchProvided
+        subs: [ 'Better Docks' ]
         condition: 'active("BetterDocks-MP-DLC-Compatible.esp") and not active("JKs Skyrim_Better Docks_Patch.esp")'
-      - <<: *hasPluginPatch
-        subs:
-          - 'Bring Out Your Dead'
-          - '[JK''s Skyrim - Arthmoor''s Patch Pack](https://www.nexusmods.com/skyrimspecialedition/mods/6289/)'
+      - <<: *patchProvided
+        subs: [ 'Bring Out Your Dead' ]
         condition: 'active("Bring Out Your Dead.esp") and not active("JKs Skyrim_BOYD_Patch.esp")  and not active("JKs Skyrim_Arthmoor All_Patch.esp")'
-      - <<: *hasPluginPatch
-        subs:
-          - 'Carriage and Ferry Travel Overhaul'
-          - '[JK''s Skyrim SE CFTO Patch](https://www.nexusmods.com/skyrimspecialedition/mods/6289/)'
+      - <<: *patchProvided
+        subs: [ 'Carriage and Ferry Travel Overhaul' ]
         condition: 'active("CFTO.esp") and not active("CFTO-JK-Patch.esp")'
-      - <<: *hasPluginPatch
-        subs:
-          - 'Dawn of Skyrim'
-          - '[JKs Skyrim_Dawn of Skyrim_Patch.esp](https://www.nexusmods.com/skyrimspecialedition/mods/16852/)'
+      - <<: *patchProvided
```

Comment:
The patch isn't at https://www.nexusmods.com/skyrimspecialedition/mods/6289/

Reply:
Fixed in https://github.com/loot/skyrimse/pull/468/commits/bbf821e16285ff6a1764b402bea99665fedb6fa4